backend/wayland/seat.c

```diff
@@ -448,7 +448,11 @@ struct wlr_wl_input_device *create_wl_input_device(
 	struct wlr_input_device *wlr_dev = &dev->wlr_input_device;
 
 	unsigned int vendor = 0, product = 0;
-	const char *name = "wayland";
+
+	size_t name_len = 8 + strlen(seat->name) + 1;
```

Comment:
I generally try to use "len" for the result of `strlen` (number of items) and "size" for the number of bytes. Since this include the final `\0` I'd be more comfortable with `name_size` here.